REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.6
-MathProgBase 0.6 0.8
-ReverseDiffSparse 0.8 0.9
-ForwardDiff 0.5 0.8
+MathProgBase 0.6
```

Comment:
Why removing the upper bounds ?

Reply:
Sorry, I forgot about this, this may be a potential problem.  In some of these cases the upper bounds are not good for 0.7.  We might have to go back and make 0.7 compatibility updates to the earlier releases of some of these packages.  I'll look into this when I get a chance.

Reply:
If we make a new version of these packages that work with v0.7, we will update the upper bounds in JuMP

Reply:
Yeah, sorry, I took those out for testing and forgot to restore them.  Will do so now.